static/_redirects

```diff
@@ -1,7 +1,9 @@
 #redirect file for moved web pages.
-/main-concepts                              /tooling
-/history                                    /research
-/cosign/overview                            /signing/quickstart    
+/main-concepts                              /about/tooling
+/security                                  /about/security
```

Comment:
This one still is one space out 🤔 

```suggestion
/security                                   /about/security
```

Reply:
Maybe I should have seen if there was an Emacs mode for weird unspecified URL lists, ha.